Extra Qualification on Member Error when compiling smconv

Got this error while compiling smconv on my Mac OS X Catalina system through the command line:
```
./it2spc.h:130:16: error: extra qualification on member 'ParseSMOption'
                void Module::ParseSMOption( const char * );
                     ~~~~~~~~^
./it2spc.h:131:16: error: extra qualification on member 'ParseSMOptions'
                void Module::ParseSMOptions( const ITLoader::Module & );
                     ~~~~~~~~^
```

My compiler from `gcc -v`...
```
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/c++/4.2.1
Apple clang version 12.0.0 (clang-1200.0.32.29)
Target: x86_64-apple-darwin19.6.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

The main source of this fault is caused by Visual Studio doing things that aren't supported by standard compliant compilers.

--- a/tools/smconv/it2spc.h
+++ b/tools/smconv/it2spc.h
@@ -127,8 +127,8 @@ namespace IT2SPC {
 		std::vector<Sample*> Samples;
 		
 
-		void Module::ParseSMOption( const char * );
-		void Module::ParseSMOptions( const ITLoader::Module & );
+		void ParseSMOption( const char * );
+		void ParseSMOptions( const ITLoader::Module & );
 		
 	public:
 		Module( 

